errbot/plugin_manager.py

```diff
@@ -356,7 +356,7 @@ def activate_non_started_plugins(self):
             try:
                 if self.is_plugin_blacklisted(name):
                     errors += f'Notice: {plugin.name} is blacklisted, ' \
-                              f'use {self.bot.prefix}plugin unblacklist {name} to unblacklist it.\n'
+                              f'use plugin unblacklist {name} to unblacklist it.\n'
```

Comment:
Based on earlier findings, this should work.
```suggestion
                              f'use "{self.plugins["Help"]._bot.prefix}plugin unblacklist {name}" to unblacklist it.\n'
```